internal/clients/wit.go

```diff
@@ -10,15 +10,19 @@ import (
 )
 
 // WIT describes work item tracker service of OSIO.
-type WIT struct {
+type WIT interface {
+	SearchCodebase(repo string) (*WITInfo, error)
```

Comment:
Do you know why does this method returns an pointer instead of a value? Is this how the WIT is defined elsewhere?

Reply:
I think that it is because it comes from an `Umarshal` call